docs/community/contributing.md

```
@@ -0,0 +1,276 @@
+### Make Changes
+
+1. Write clean, readable code
+2. Follow the existing code style
+3. Add XML documentation for public APIs
+4. Include unit tests for new functionality
+
+### Commit Guidelines
+
+Write clear commit messages:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)